App_Programmering/Produkt_beskrivelse.md

```diff
@@ -0,0 +1,17 @@
+# Produkt beskrivelse
+
+__Hvilken form for app:__
+Jeg kunne godt bare tænke mig at udvilke mig fagligt, i forhold til at prøve lidt af hvert. Kan ikke helt beslutte mig for om jeg vil udvikle i React eller Android studio.
+Men min idé er at lave en "shelf" menu, hvor der er en masse muligheder for at tilføje forskellige tabs til forskellige features.
+
+__Features:__
+
+- Dark-theme
+- Mulighed for at oprette en lokal profil
+    - Brug af kamera i forhold til at uploade profil billede
+        - Finde en måde hvor man kan beskære billeder osv
+    - Måske mulighed for noget api til en database
```

Comment:
Vil nok lade det her med database ligge for nu. Vigtigere at fokusere på selve app delen.